src/openrct2/paint/tile_element/Paint.LargeScenery.cpp

```diff
@@ -289,8 +289,7 @@ void PaintLargeScenery(paint_session* session, uint8_t direction, uint16_t heigh
     boxlength.x = s98E3C4[esi].length.x;
     boxlength.y = s98E3C4[esi].length.y;
     boxlength.z = boxlengthZ;
-    PaintAddImageAsParent(
-        session, image_id, 0, 0, boxlength.x, boxlength.y, boxlengthZ, height, boxoffset.x, boxoffset.y, boxoffset.z);
+    PaintAddImageAsParent(session, image_id, { 0, 0, height }, { boxlength.x, boxlength.y, boxlengthZ }, boxoffset);
```

Comment:
The line immediately above this already creates a CoordsXYZ for boxlength.

```suggestion
    PaintAddImageAsParent(session, image_id, { 0, 0, height }, boxLength, boxoffset);
```